src/components/Sum.jsx

```diff
@@ -1 +1,7 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({ a = 0, b = 0 }) => (
+  <p>
+    {`Sum of ${a} and ${b} is ${Number(a) + Number(b)}`}
```

Comment:
```suggestion
    {`Sum of ${a} and ${b} is ${a + b}`}
```
I think there is no need to convert a and b into numbers, as you pass only numbers